prow/gerrit/adapter/adapter.go

```diff
@@ -427,6 +427,19 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 		logger.WithField("duration(s)", time.Since(tracker).Seconds()).Debug("Time taken for gerrit change to be processed.")
 	}()
 
+	if change.Revisions != nil {
+		if rev, ok := change.Revisions[change.CurrentRevision]; ok {
```

Comment:
Prow polls Gerrit periodically, and there could be more than 1 revision in between polls, a more reliable way would be to figure out all the changes since last poll, I think `LastSyncState` is what this is used for

Reply:
Makes sense, done.